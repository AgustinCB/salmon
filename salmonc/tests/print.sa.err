Constants: [Integer(1), Integer(3), String(0), String(5)]
Instructions: [Instruction { instruction_type: Constant(2), location: 0 }, Instruction { instruction_type: Push, location: 0 }, Instruction { instruction_type: Strlen, location: 0 }, Instruction { instruction_type: Swap, location: 0 }, Instruction { instruction_type: Constant(0), location: 0 }, Instruction { instruction_type: Constant(1), location: 0 }, Instruction { instruction_type: Constant(0), location: 0 }, Instruction { instruction_type: Syscall, location: 0 }]
Locations: [Location { address: 5, line: 0 }]
