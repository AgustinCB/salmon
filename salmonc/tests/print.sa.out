HOLA
0 - Integer(5)
